Tests/FoundationTests/NSAttributedStringExtensionsTests.swift

```diff
@@ -306,6 +306,24 @@ final class NSAttributedStringExtensionsTests: XCTestCase {
         #endif
     }
 
+    // MARK: - Initializer
+    
+    func testInitImage() {
+        #if canImport(UIKit)
```

Comment:
The `#if` in the tests should match exactly with what's in the implementation of the function. And it should be outside the function, not inside.